src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -66,7 +71,7 @@ private void Traverse(JsonSchema schema, JToken token, Action<JsonSchema, JValue
                     {
                         if (schema.Properties.TryGetValue(key, out var propertySchema))
                         {
-                            Traverse(propertySchema, value, transformScalar);
+                            Traverse(propertySchema, value, transformScalar, schema);
```

Comment:
If you pass `key` down to traverse, than the complex `GetPropertyNameFromJsonPath` is not needed
```suggestion
                            Traverse(propertySchema, value, key, transformScalar, schema);
```

Reply:
don't want to do that  since it's change too much of Traverse and GetPropertyNameFromJsonPath is also used by XrefMapBuilder